packages/block-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "block-scripts",
-  "version": "0.0.7",
+  "version": "0.0.8-next.1",
```

Comment:
What is the reason for using `next`? This creates some confusion between ‘next version’ and Next.js. which we also use. I refer to confusion in conversations rather than the code itself. It’s nice to have a technical vocabulary where each word has only one meaning.

Reply:
Per your suggestion [here](https://github.com/blockprotocol/blockprotocol/pull/336#discussion_r893702695)

Reply:
We can use `canary` in future if you now would prefer not to use `next`. I suggest we don't do this now as it will mean we have even more tags in published packages.